src/test/java/com/easypost/AddressTest.java

```diff
@@ -81,8 +83,14 @@ public void testCreateVerify() throws EasyPostException {
         assertEquals("417 MONTGOMERY ST FL 5", address.getStreet1());
 
         assertNotNull(address.getVerifications());
-        assertNotNull(address.getVerifications().getZip4().getErrors()); // Should have a error due to second line
-        assertNotNull(address.getVerifications().getDelivery().getErrors());
+
+        assertFalse(address.getVerifications().getDelivery().getErrors().isEmpty()); // should have at least one error
```

Comment:
suggestion: the double negative is hard to read. We should probably be asserting that it is present instead

Reply:
IDE didn't like that we weren't using this "simplified" way.